src/commands/Mafia/nightmessages.js

```diff
@@ -0,0 +1,186 @@
+const _ = require('lodash');
+// to add options/change gui: simply add/change this
+let gui = {
+    reset      : {emoji: '♻',  text: 'Reset the content of this message\n'},
+    peace      : {emoji: '☮',  text: 'You slept peacefully.\n'},
+    custom     : {emoji: '📧', text: 'Click this to add a custom message.\n'},
+    roleblocked: {emoji: '🚫', text: 'You got rolebocked last night.\n'},
+    jailed     : {emoji: '🏰', text: 'You were jailed last night.\n'},
+    controlled : {emoji: '💫', text: 'You got controlled by a Witch last night.\n'},
+    mafia_kill : {emoji: '💀', text: 'You got killed by the Mafia last night.\n'},
+    sk_kill    : {emoji: '🔫', text: 'You got killed by the Serial Killer last night\n'},
+    immune     : {emoji: '👎', text: 'Your target was immune to your attack.\n'},
+    bodyguarded: {emoji: '💪', text: 'Your target was protected by the Bodyguard and you were killed by them last night.\n'},
+    send       : {emoji: '✅',  text: '\n**You have 30 minutes to use this menu then it will time out!\n' +
+                                     'If you are satisfied with the messages click the  ✅ below this message to send all (not empty) messages out.\n' +
+                                     'To cancel click the 🇽**'},
+    abort      : {emoji: '🇽', text: ''},
+
+    // get the correct text to an emoji
+    text_for_emote: function (emoji_name) {
+        return Array.from(gui).find(elem => {if (elem.emoji === emoji_name) return elem;}).text;
+    },
+
+    // construct text for displaying a legend of which emoji does what
+    legend_text: function () {
+        return '\n\n\n**Legend:**\n' + Array.from(gui).map(elem => {return elem.emoji + ' : ' + elem.text;}).join('') + this.send.text;
+    },
+
+    // Iterator (ignores the 'send' emote keep that in mind)
+    [Symbol.iterator]: function*() {
+        for (let key in this) {
+            if (this.hasOwnProperty(key) && typeof(this[key]) !== 'function' && key !== 'send' && key !== 'abort') yield this[key];
+        }
+    }
+};
+let msgs, alive_players;
+
+exports.run = function (bot, msg) {
+    if (!msg.member.hasPermission('ADMINISTRATOR') && !bot.mafia.isMod(msg.author.id)) {
+        msg.channel.send(':negative_squared_cross_mark:  |  You are not a game moderator.');
+        return;
+    }
+
+    // The node.js default is 10 open listeners... I need one for each person + one for sending/canceling will be reset to 10 again at the end
+    process.setMaxListeners(0);
```

Comment:
yep 100 seem feasible (it will never even get close to that and they have a timeout and all but better safe than sorry )